client/packages/common/src/intl/utils/IntlUtils.ts

```diff
@@ -62,28 +49,47 @@ export const IntlUtils = {
       throw new Error(`Language '${language}' not supported`);
     }
     return 'en';
-  },
-  languageOptions,
-  getLanguageName: (language: string) =>
-    languageOptions.find(option => option.value === language)?.label,
```

Comment:
Usage only ever returned the current language `getLanguageName(i18n.language)`, so makes it simpler to just have the hook return `currentLanguage` as a value.